librz/core/cmd/cmd.c

```diff
@@ -3537,10 +3542,27 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(help_stmt) {
 	return res;
 }
 
+static uint32_t tmp_child_idx = 0;
+
+static TSNode tmp_get_next_node(TSNode cur) {
+	TSNode next = ts_node_next_named_sibling(cur);
+	tmp_child_idx++;
+	if (ts_node_is_null(next)) {
+		next = ts_node_named_child(ts_node_parent(cur), 0);
+		tmp_child_idx = 0;
+	}
+	return next;
+}
+
```

Comment:
I don't like this tbh, it makes parallel parsing of commands not safe.

Reply:
If we ever, ever decide to parse commands in parallel, I think the usual tricks can be applied e.g. make `tmp_child_idx` a thread-local variable, pass `tmp_child_idx` around as an argument etc.